addons/lunch/models/lunch.py

```diff
@@ -74,22 +74,6 @@ def _compute_get_previous_order_ids(self):
             for order in prev_order
         }.values()
 
-    @api.onchange('total')
-    def _onchange_total(self):
```

Comment:
why remove this onchange method? now Insufficient balance information is display on bootstrap alert box instead of open warning dialog box.

may be need to re-base branch with master.